### ISSUE TITLE
Review OCP 4.11 Release Notes - William

### ISSUE THREAD

**Issue by tbaker1313** [BCDevOps/developer-experience#3414]
**Describe the issue**
Review the [OCP 4.11 Release Notes](https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html) for any changes that will impact the product teams, ops team, or platform services team. Also look for any exciting new features that community should know about.

**What is the Value/Impact?**
Ensure the product teams are kept informed about changes.

**What is the plan? How will this get completed?**
Read the release notes. Add any bits you find interesting and needing of discussion as comments here.

**Identify any dependencies**
None

**Definition of done**
Fully read the release notes and added anything of interest as a comment here.

**Comment by wmhutchison** [comment]
Will backport anything new or needing adjusting in the parent ticket description field.

**Updates of Interest for the Community**
* Service account tokens no longer being generated by default definitely should be included in the email comms for the community.
* the SCC changes will likely take front stage as well for email comms for the developers/community. Both for the specific use case discovered regarding permissions when putting executable scripts in ConfigMap. Also the general work-around available once we've prepped the cluster for cases where the new SCC might cause general failure for some workloads.

The [Bugfixes](https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-bug-fixes) section of the OCP 4.11 notes doesn't allow further linking to be more specific, but has a literal tonne bugfixes for all sorts of things, including things of possible interest to the community. Would recommend at best just putting a link out for the community to explore if desired.

**Updates for Internal Interest**
* general support for RHEL9 allows us to proceed with UTIL server refreshes this year since RHEL9 will be GA I believe this spring?
* tuning options for max connections allowed per haproxy router instance. Not sure if we want/need to mess with this, but if we do, the option's now there.
* Not used at present, but if we were to install the descheduler operator, it now offers a dry-run mode where it'll tell us what pods it'd remove if allowed, but not actually do it. Curious what it'd do with some of our more problematic nodes. 
* MCO changes will now warn us when logged into the web console if we've got a machine config pool paused and there's a need to push new certs to said paused pool. Not really a big deal since we normally only do the mcp pause game during OCP upgrades, but helpful in warning us of potential breakage if we decide to pause pools for longer periods and not during OCP upgrades. My take on how it alerts is likely just AlertManager, meaning we'd get an email as well about it.

**Comment by wmhutchison** [comment]
Will be updating the parent EPIC's main docs for the following item so that it's not under "for admins" but under "For developers" since the effects of this change will apply to all applications hosted in the user projects/namespaces:

```
The haproxy.router.openshift.io/balance variable, which sets the router load-balancing algorithm, now defaults to the value random instead of leastconn.
```
